### PR TITLE
continuous input; allow exit on 'q' or 'Q'

### PR DESCRIPTION
**Pull request by echolabstech**
- changed the way input works so its continuously reading each new line of text
- added controls to quit the app using 'q' or 'Q' key

### CODE REVIEW

File: Palindrome Algorithm_01.js [hunk 1]
```diff
@@ -18,19 +18,18 @@ function checkPalindrome(a){ //check if the String a is a palindrome. returns st
...
-//var input = [];
 
 var readline = require('readline');
 var rl = readline.createInterface(process.stdin, process.stdout);
+console.log("input text: ");
+rl.on('line', (line) => {
+	if (line.toLowerCase() === 'q') rl.close();
 
+  if (isPalindrome(line)) {
+  	console.log(`"${line}" is a palindrome :-)`)
```
Comment (KangVictor) [inline Palindrome Algorithm_01.js]: what is ${} sign?

File: Palindrome Algorithm_01.js [hunk 2]
```diff
@@ -1,10 +1,9 @@
-function checkPalindrome(a){ //check if the String a is a palindrome. returns string test "true" if so.
+function isPalindrome(a){ //check if the String a is a palindrome. returns string test "true" if so.
 	var b = a.split('');// split the string into array of char
 
 	if(b.length % 2 == 0) {// if even number of letters
 		for(var n = 0; n < b.length/2; n++) {
 			if (b[n] != b[b.length - n - 1]) {
-				return 'false';
```
Comment (KangVictor) [inline Palindrome Algorithm_01.js]: **Blocker**
The algorithm won't work without this code.
[https://gist.github.com/KangVictor/a8293bcf6403f4569a707802d52c1e14](url)

Comment (KangVictor) [reply Palindrome Algorithm_01.js]: this line

File: Palindrome Algorithm_01.js (hunk 1 above)
Comment (echolabstech) [reply Palindrome Algorithm_01.js]: short answer, [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

File: Palindrome Algorithm_01.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-function checkPalindrome(a){ //check if the String a is a palindrome. returns string test "true" if so.
+function isPalindrome(a){ //check if the String a is a palindrome. returns string test "true" if so.
 	var b = a.split('');// split the string into array of char
 
 	if(b.length % 2 == 0) {// if even number of letters
```
Comment (KangVictor) [inline Palindrome Algorithm_01.js]: [**A change** (PR comments with a purpose)](https://gist.github.com/echolabstech/a905ad02efba1ef6663c96ee4f251480):The return type is a string, not Boolean

**Pull request merged**